test/assets/messages/WhatDoYouWantToDoMessages.scala

```diff
@@ -20,10 +20,10 @@ object WhatDoYouWantToDoMessages extends BaseMessages {
 
   val title: String = "What do you want to do?"
   val h1: String = "What do you want to do?"
-  val leftOptionMsg: String = "Start a new underpayment disclosure"
-  val rightOptionMsg: String = "Add more information to an existing underpayment disclosure"
-  val leftOptionHint: String = "Tell us about an underpayment of Customs Duty or import VAT."
-  val rightOptionHint: String = "You will need the disclosure reference number."
+  val createOptionMsg: String = "Start a new underpayment disclosure"
+  val updateOptionMsg: String = "Add more information to an existing underpayment disclosure"
+  val cancelOptionMsg: String = "Cancel an existing underpayment disclosure"
+  val createOptionHint: String = "Tell us about an underpayment of Customs Duty or import VAT."
   val error: String = "User must select an option"
```

Comment:
then here too

Reply:
Also could you change these to be "CreateCase", "UpdateCase", "CancelCase" instead of Option?